src/components/JournalPreview/index.jsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import JournalPage from '../JournalPage';
+
+class JournalPreview extends React.Component {
+  componentDidMount() {
+    this.props.getPreview(this.props.match.params.previewId);
+  }
+  componentDidUpdate(prevProps) {
```

Comment:
NIT: This isn't doing any harm, but we probably don't need this method unless we'll ever navigate from one preview to another in the same app load. It's fine to leave it if that's a potential use case.